src/Groups/GroupDetails/GroupDetails.js

```diff
@@ -51,6 +52,8 @@ const GroupDetails = ({
   const accordionStatusRef = useRef();
   const history = useHistory();
 
+  const { perms, isLoading: isPermsLoading } = useAcqRestrictions(group.id, group.acqUnitIds);
```

Comment:
that word `perms` might be confused with actual permissions. IMHO, we should clearly differentiate these

Reply:
mb restrictions as I used in hook name